### PR TITLE
[Bug 743360] Added function to retrive the last searched term

### PR DESCRIPTION
**Pull request by anushbmx**
This is fill the search field with previous searched team if exist instead of the placeholder.

### CODE REVIEW

File: kitsune/search/views.py [hunk 1]
```diff
@@ -266,6 +266,10 @@ def advanced_search(request, template=None):
     # 2. Build form.
     search_form = AdvancedSearchForm(r, auto_id=False)
     search_form.set_allowed_forums(request.user)
+    # get value for search input from last search term.
+    last_search = request.COOKIES.get(settings.LAST_SEARCH_COOKIE)
+    if last_search:
+    	r['q'] = urlquote(last_search)
```
Comment (mythmon) [inline kitsune/search/views.py]: Shouldn't you only set `r['q']` in the case that it is empty, to avoid overriding other searches?

**Comment by anushbmx** [comment]
@mythmon I have added a check for that too.

File: kitsune/search/views.py [hunk 2]
```diff
@@ -266,6 +266,10 @@ def advanced_search(request, template=None):
     # 2. Build form.
     search_form = AdvancedSearchForm(r, auto_id=False)
     search_form.set_allowed_forums(request.user)
+    # get value for search input from last search term.
+    last_search = request.COOKIES.get(settings.LAST_SEARCH_COOKIE)
+    if last_search and not 'q' in r:
```
Comment (mythmon) [inline kitsune/search/views.py]: Instead of `not 'q' in r` this is usually written as `'q' not in r`.

**Comment by anushbmx** [comment]
@mythmon I have updated it.

**Pull request merged**